### PR TITLE
new network init routine

### PR DESCRIPTION
**Pull request by lschellenberg**
this init routine targets the new structure in repo network-configs.

It also adds the possibility to have different clients than prysm

### CODE REVIEW

File: src/network/l16_node_config.go [hunk 1]
```diff
@@ -24,14 +24,14 @@ var DefaultL16NodeConfigs = &NodeConfigs{
 		Etherbase:    "0x7781121fd00A009670E31b76A2bf99b3A2D6878D",
 		DataVolume:   "./data/execution_data",
 		NetworkId:    "83748374",
-		Bootnode:     "enode://c2bb19ce658cfdf1fecb45da599ee6c7bf36e5292efb3fb61303a0b2cd07f96c20ac9b376a464d687ac456675a2e4a44aec39a0509bcb4b6d8221eedec25aca2@35.204.5.43:30303",
+		Bootnode:     "enode://c2bb19ce658cfdf1fecb45da599ee6c7bf36e5292efb3fb61303a0b2cd07f96c20ac9b376a464d687ac456675a2e4a44aec39a0509bcb4b6d8221eedec25aca2@34.91.51.22:30303\n",
```
Comment (atif-konasl) [inline src/network/l16_node_config.go]: Our current geth bootnodes enode address:

bootnode-0:
`enode://c2bb19ce658cfdf1fecb45da599ee6c7bf36e5292efb3fb61303a0b2cd07f96c20ac9b376a464d687ac456675a2e4a44aec39a0509bcb4b6d8221eedec25aca2@34.90.85.198:30303`

bootnode-1:
`enode://c2bb19ce658cfdf1fecb45da599ee6c7bf36e5292efb3fb61303a0b2cd07f96c20ac9b376a464d687ac456675a2e4a44aec39a0509bcb4b6d8221eedec25aca2@34.91.166.253:30303`

bootnode-2:
`enode://c2bb19ce658cfdf1fecb45da599ee6c7bf36e5292efb3fb61303a0b2cd07f96c20ac9b376a464d687ac456675a2e4a44aec39a0509bcb4b6d8221eedec25aca2@34.91.51.22:30303`

Please use any of them and there is newline(`\n`) at the end of the enode address

File: src/network/l16_node_config.go [hunk 2]
```diff
@@ -24,14 +24,14 @@ var DefaultL16NodeConfigs = &NodeConfigs{
...
 	},
 	Consensus: &ClientDetails{
 		StatsAddress: "34.141.143.70:9090",
 		Verbosity:    "info",
-		Version:      "v0.2.1-dev",
+		Version:      "v0.2.2-dev",
 		DataVolume:   "./data/consensus_data",
-		Bootnode:     "enr:-MK4QOoOAELWWC0dZ7hwZzDY3NhxbGJWB9JFBGsIswzF383NRPNh7vfI_K4gt5KMCFt6-NrMbUdizURmcKE5xjfRhBaGAYBwAjI7h2F0dG5ldHOIAAAAAAAAAACEZXRoMpAMEg0LYQAAcAMAAAAAAAAAgmlkgnY0gmlwhCPMBSuJc2VjcDI1NmsxoQO4XSsbls7lyhfqvcsgS8jmjFmBpC3dekXssvAEXkHtJYhzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A",
+		Bootnode:     "enr:-MK4QDjZAzD165YMiCM6WrCdRN3EGDViJqP_v1fyOE-YuOYAQ102NnF8GjTJG_-yUUMwWKuEzRMJMIoObbgDB3jtZA6GAYCJeizfh2F0dG5ldHOIAAAAAAAAAACEZXRoMpCBQMXLYgAAcf__________gmlkgnY0gmlwhCJbpv2Jc2VjcDI1NmsxoQOGOP29TdRKCr40kT-ZgsvrMYIs6EIwuYGEq11txAj_VohzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A",
```
Comment (atif-konasl) [inline src/network/l16_node_config.go:34]: Our current beacon bootnodes enr address:
bootnode-0:
`enr:-MK4QIPHnsUAHhMfYvSfSsuskYeWbrYEF7u4RWwhKiNzff76FoR8nqGUa6zNMtHdQnq3jyGdhpsy_-mssSRyyNUDfLOGAYCJeizuh2F0dG5ldHOIAAAAAAAAAACEZXRoMpCBQMXLYgAAcf__________gmlkgnY0gmlwhCJaVcaJc2VjcDI1NmsxoQP2qSmV-Ga7QOfq3svJAHIf-0knhdssBTkqZwJANTyIoIhzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A`

bootnode-1:
`enr:-MK4QDjZAzD165YMiCM6WrCdRN3EGDViJqP_v1fyOE-YuOYAQ102NnF8GjTJG_-yUUMwWKuEzRMJMIoObbgDB3jtZA6GAYCJeizfh2F0dG5ldHOIAAAAAAAAAACEZXRoMpCBQMXLYgAAcf__________gmlkgnY0gmlwhCJbpv2Jc2VjcDI1NmsxoQOGOP29TdRKCr40kT-ZgsvrMYIs6EIwuYGEq11txAj_VohzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A`

bootnode-2:
`enr:-MK4QMAx93vdciWge1Od_GwaXafBh2I3Sct-VWUsuiQufP6nCEbJTzctfADuhQ8QHzC5ytSTPAfB7v-y6eT3YqxlRJSGAYCJeizYh2F0dG5ldHOIAAAAAAAAAACEZXRoMpCBQMXLYgAAcf__________gmlkgnY0gmlwhCJbMxaJc2VjcDI1NmsxoQOoiAXiE-kUiL8Qw8R820U4Ce1laLWDk5hQ6UGy_jO4yYhzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A`

Please use any of them

**Review by atif-konasl — CHANGES_REQUESTED** [review]

File: src/network/l16_node_config.go (hunk 1 above)
Comment (lschellenberg) [reply src/network/l16_node_config.go]: yes, i used the second and will remove the newline

File: src/network/l16_node_config.go (hunk 2 above)
Comment (lschellenberg) [reply src/network/l16_node_config.go:34]: I used the first one

**Pull request merged**